src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  loadingTodos: number[];
+  deleteTodo: (todoId: number) => void;
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  todo,
+  loadingTodos,
+  deleteTodo,
+}) => {
+  const {
+    id, title, completed,
+  } = todo;
+
+  const handleDeleteButton = () => {
+    deleteTodo(id);
+  };
+
+  return (
+    <div className={cn('todo', { completed })}>
+      <label className="todo__status-label">
+        <input
+          type="checkbox"
+          className="todo__status"
+          checked={completed}
+        />
+      </label>
+
+      <span className="todo__title">{title}</span>
+
+      <button
+        type="button"
+        className="todo__remove"
+        onClick={handleDeleteButton}
+      >
+        ×
+      </button>
+
+      <div className={cn('modal overlay', {
+        'is-active': loadingTodos.includes(id),
```

Comment:
create variable with descriptive name for `loadingTodos.includes(id)`
```suggestion
        'is-active': loadingTodos.includes(id),
```